src/cogs/misc/main.py

```diff
@@ -1,4 +1,5 @@
 from datetime import datetime, timedelta
+import dateutil.parser
```

Comment:
Why was this imported?

Reply:
Oh forgot to remove that one, it was from when i tried parsing real date and times, other than a number of seconds